pkg/apiclient/workflow/workflow.proto

```diff
@@ -147,6 +148,10 @@ service WorkflowService {
     option (google.api.http).get = "/api/v1/workflows/{namespace}/{name}";
   }
 
+  rpc ListK8SAndArchivedWorkflows(WorkflowListRequest) returns (github.com.argoproj.argo_workflows.v3.pkg.apis.workflow.v1alpha1.WorkflowList) {
+    option (google.api.http).get = "/api/v1/k8s-archived-workflows/{namespace}";
+  }
+
```

Comment:
So I was thinking that this would work more roughly within the constraints of the v3.5 API, as in, implementing it as part of the `workflows` API method just as optional parameters. Would that not work?

Reply:
I also kept thinking about that part.
If so, I will reflect this by branching to parameters within `ListWorkflows` and separating them.

Thank you for your feedback!